### DISCUSSION TITLE
Running in port 587 .  Port 25 blocked by my ISP.

### DISCUSSION THREAD

**Discussion by Sylbo** [huan/docker-simple-mail-forwarder#28] [General]
Amazon Lightsail Cloud service doesn't allow to open the 25 port.
Can I use the image on another SMTP standard port, like 587?

I tried running the image:
	docker run -t -d -e SMF_CONFIG="$SMF_CONFIG" -p 587:25 zixia/simple-mail-forwarder
I have a response from telnet, but forwarding doesn't seem to work.

Is it possible to run the image this way?

Thanks!

**Comment by huan** [comment]
Hi @Sylbo ,

I have never setup an SMTP server on non-standerd port.

Hope there could be other people could help you.

Good luck!

**Comment by SamMousa** [comment]
@Sylbo your configuration should work fine with respect to port forwarding.
There could indeed be an issue with how postfix sees the "remote IP" of the connection.
This depends on the type of networking you're using.
In your postfix logs you should see which IP is being blocked.
If you see an IP that is from your local machine you could allow it.

For example, if your docker containers get IPs in the range `10.42.0.0/24`, since that is not an internet routable range you should be fine allowing those IPs to be forwarded via `SMF_MYNETWORKS`.

If you're not sure what you're doing be sure to do an online SMTP open proxy test!

Good luck.
